src/jit/gentree.h

```diff
@@ -3284,6 +3284,9 @@ struct GenTreeCall final : public GenTree
 #define GTF_CALL_M_TAILCALL_VIA_HELPER 0x0200 // GT_CALL -- call is a tail call dispatched via tail call JIT helper.
 #endif                                        // !LEGACY_BACKEND
 
+#define GTF_CALL_M_FAT_POINTER_CHECK 0x0200 // GT_CALL -- it is calli that needs to be tranformed
+                                            // to check CoreRT fat pointer case.
```

Comment:
This flag has the same value as that of GTF_CALL_M_TAILCALL_VIA_HELPER.  It would mean we are not expecting calli to be never dispatched as a tail call via helper.
This might be true now but it would be a good idea to use a different value for it. (e.g. 0x0400)